draft-ietf-acme-acme.md

```diff
@@ -1972,6 +1997,11 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the server matches the token for
    this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
```

Comment:
Actually, MAY matches the [HTTP RFC](https://tools.ietf.org/html/rfc7231#section-6.4):

> If a Location header field (Section 7.1.2) is provided, the user agent MAY automatically redirect its request to the URI referenced by the Location field value